src/main/java/com/example/demo/service/BulkyGarbageFacilityService.java

```diff
@@ -0,0 +1,9 @@
+package com.example.demo.service;
+
+import com.example.demo.model.BulkyGarbageFacilityDTO;
+
+import java.util.List;
+
+public interface BulkyGarbageFacilityService {
+    List<BulkyGarbageFacilityDTO> fetchFacilities(Integer id);
```

Comment:
```suggestion
    List<BulkyGarbageFacilityDTO> searchFacilities(Integer id);
```

DBにアクセスして想定する取得件数に応じてのメソッド名の付け方（Service上の話）は以下の通り。
- 1件: `get*` または `retrieve*`
- 複数件（Listで返却する）: `search*`

Reply:
1件の場合はget、複数県の場合はsearchに統一しました